Sources/Basic/FileSystem.swift

```diff
@@ -116,7 +117,7 @@ public enum FileMode {
 /// natural filesystem interface, while still allowing clients to transparently
 /// substitute a virtual file system or redirect file system operations.
 ///
-/// NOTE: All of these APIs are synchronous and can block.
+/// - NOTE: All of these APIs are synchronous and can block.
```

Comment:
Should this be "Note:" instead as the other ones?

Reply:
`- Note:` is highlighted by Xcode and so important topics are not missed by a consumer. But yea it should be `- Note:` instead of `- NOTE:` I will fix that.